static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -74,15 +74,23 @@ function useExtractedCrumbHtml({replay}: HookOpts) {
             if (isMounted) {
               setBreadcrumbReferences(rows);
             }
-            document.body.removeChild(domRoot);
+            setTimeout(() => {
+              if (document.body.contains(domRoot)) {
+                document.body.removeChild(domRoot);
+              }
+            }, 0);
```

Comment:
add a `setTimeout` so any tasks created by the `new Replayer` instance have a chance to finish before we rip the node out of the document.